templates/focus-ios.xliff

```diff
@@ -557,6 +658,18 @@
         <source>Example: searchengineexample.com/search/?q=%s</source>
         <note>Text displayed as an example of the template to add a search engine.</note>
       </trans-unit>
+      <trans-unit id="share">
+        <source>Share</source>
+        <note>Text for a share button</note>
+      </trans-unit>
+      <trans-unit id="share.trackerStatsLabel">
+        <source>%@ trackers blocked so far</source>
+        <note>Text used when the user shares their trackers blocked stats</note>
+      </trans-unit>
+      <trans-unit id="share.trackerStatsText">
+        <source>%1$@, the privacy browser from Mozilla, has already blocked %2$@ trackers for me. Fewer ads and trackers following me around means faster browsing! Get Focus for yourself here</source>
+        <note>The text shared to users after the user chooses to share there tracker stats</note>
```

Comment:
Same comment about both placeholders.
Also not: in the note it says "there" --> should be "their"

Reply:
As above, this won't work with languages using complex plural forms.

Reply:
Would love @brjones opinion on how to word this text that can be localized in a clean manner.